pkg/cmd/adm/unregister_member_test.go

```diff
@@ -65,9 +65,9 @@ func TestUnregisterMemberCallsRestart(t *testing.T) {
 	ctxAct := clicontext.NewCommandContext(term, newClient)
 	called := 0
 	// when
-	err := UnregisterMemberCluster(ctxAct, "member1", func(ctx *clicontext.CommandContext, restartClusterName string) error {
+	err := UnregisterMemberCluster(ctxAct, "member1", func(ctx *clicontext.CommandContext, restartClusterName string, cfcGetter ConfigFlagsAndClientGetterFunc) error {
```

Comment:
same here , the cfGetter is not used
```suggestion
	err := UnregisterMemberCluster(ctxAct, "member1", func(ctx *clicontext.CommandContext, restartClusterName string, _ ConfigFlagsAndClientGetterFunc) error {
```

Reply:
yes right, updated it [a05065e](https://github.com/kubesaw/ksctl/pull/90/commits/a05065ec42b65641228054641e05b47a2fe10450)